Gopkg.lock

```diff
@@ -0,0 +1,198 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
+
+
+[[projects]]
+  name = "github.com/Infoblox-CTO/ngp.api.toolkit"
```

Comment:
I think reference here should point to `github.com/infobloxopen/atlas-app-toolkit`

Reply:
The app relies on `protoc-gen-gorm` plugin, which currently depends on the private version. This will be fixed in the next release of the plugin, will update `contacts-app` accordingly.